src/coreclr/gc/gc.cpp

```diff
@@ -49307,3 +49377,294 @@ void PopulateDacVars(GcDacVars *gcDacVars)
 #endif // MULTIPLE_HEAPS
     gcDacVars->generation_field_offsets = reinterpret_cast<int**>(&generation_field_offsets);
 }
+
+void GCHeap::RefreshMemoryLimit()
+{
+    gc_heap::refresh_memory_limit();
+}
+
+void gc_heap::configure_memory_settings(bool is_initialization, uint32_t& nhp, uint32_t& nhp_from_config, size_t& seg_size_from_config)
```

Comment:
since you don't actually change the value of ndp_from_config you don't need to pass by ref. for seg_size_from_config I would probably rename it to just seg_size and add a note that says on entry this is the size we get from config.